### DISCUSSION TITLE
Passwords and sensitive data is written to [INFO] in the log

### DISCUSSION THREAD

**Discussion by dimakovalenko-united-ctm** [open-webui/open-webui#6412] [Issues]
Severity: Minor

# Bug Report
 
Watching the logs, I've noticed that passwords and other sensitive issue is written into [INFO] not [DEBUG] which means I'll have to scrape logs in my production system.

``` 
INFO  [open_webui.apps.webui.models.auths] insert_new_auth
name='test2' email='test2@test.com' password='123456' profile_image_url='/user.png' role='user'
```

## Installation Method

Docker Web UI & Ollama

## Environment

- **Open WebUI Version:** 0.3.13
- **Ollama (if applicable):** [e.g., v0.2.0, v0.1.32-rc1]

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

**Confirmation:**

- [ Y ] I have read and followed all the instructions provided in the README.md.
- [ N ] I am on the latest version of both Open WebUI and Ollama.
- [ N ] I have included the browser console logs.
- [ Y ] I have included the Docker container logs.
- [ Y ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

[Describe what you expected to happen.]

## Actual Behavior:

[Describe what actually happened.]

## Description

**Bug Summary:**
Need to remove sensitive data from logs

**Comment by tkg61** [comment]
Your version of openwebui is pretty old, have you considered upgrading? i don't believe i see password info in my 0.3.32 version debug log

**Comment by dimakovalenko-united-ctm** [comment]
Digging through latest version now, will close the bug if it’s happy

**Comment by dimakovalenko-united-ctm** [comment]
This seems to be the culprit on new user creation

https://github.com/dimakovalenko-united-ctm/open-webui/blob/main/backend/open_webui/apps/webui/models/auths.py#L103
